docs/frc/5-Advanced_&_Misc/Servos.md

```diff
@@ -1 +1,40 @@
-# Servos
\ No newline at end of file
+# Servos
+
+## Overview
+
+**Servomotors**, or servos for short, are useful machines **that provide low-power, repeatable motion.** It functions similarly to a motor, as it drives an axle to induce angular motion. However, the main differences between the two are that a servo is *extremely* low-power compared to a motor, and is also coupled to a sensor with motor position feedback, or in other words, is capable of precise angular and linear position control. Because a servo's power is so weak, though, it is not effective in every situation, and that is why we don't see servos used extensively on our robots.
+
+### Examples
+
+![Hitec HS-322HD Servo](img/hitecservo.jpg ':size=250x250') ![REV Smart Servo & Programmer](img/revsmartservo.png ':size=290x250')
+
+In the left photo is the Hitec HS-322HD Servo, which was provided in the FIRST Kit of Parts in 2008. On 1257, though, we generally use the [REV Smart Servo](https://www.revrobotics.com/rev-41-1097/), pictured in the right. The large square device is called the "Servo Programmer." We can work with the electronics' members and use the programmer to customize the servo's range of motion. 
+
+A special feature that the REV Smart Servo offers is the ability to switch parameter modes, hence the "Smart" in the name. The servo is in position mode by default, and we would pass in values ranging from 0.0 to 1.0, with 0.0 and 1.0 corresponding to fully leftward and fully rightward respectively. But, we can use the Servo Programmer to switch the servo into speed mode and use it like a regular motor by passing in a speed value in between -1 and 1. We would use the Servo Programmer to define those -1 and 1 speed thresholds ourselves.
+
+## Usage Cases
+
+![Servo Intake Up](img/intakeup.jpg ':size=350x560') ![Servo Intake Down](img/intakedown.jpg ':size=325x560')
+
+For 1257's 2020 robot, we used a polycarbonate clip attached to a servo to hold up the roller intake and keep it in the robot's frame perimeter, as labeled in the left photo. We planned to drive the servo clip up at the beginning of autonomous so that the intake would be left down for the rest of the match, like it is in the right photo. This is just one example of the many applications a servo has to offer. 
+
+You might be thinking that it would've been easier to add a motor to the intake and adjust it mechanically so that it would be rotatable/jointed (allowing us to simply lower the intake manually at the beginning of a match), but we already had numerous motors on the robot. Adding that extra motor would have really strained the current draw on the battery and possibly could have introduced other issues. Thus, a servo was the most suitable solution for keeping the intake within frame perimeter. 
```

Comment:
You can also mention how the servo's built in closed loop functionality also makes it trivially easy to program this and ensure we don't slam the hook into the indexer, which is a possibility with a continuous motor.